MODULE.md

```diff
@@ -0,0 +1,262 @@
+# Modules
+
+## Table of Contents
+
+- [Modules](#modules)
+  - [Manifest Module](#manifest-module)
+    - [Module Functionality](#module-functionality)
+      - [Asset Issuance](#asset-issuance)
+      - [Commands](#commands)
+        - [Update Parameters (update-params)](#update-parameters-update-params)
+        - [Stakeholder Payout (stakeholder-payout)](#stakeholder-payout-stakeholder-payout)
+  - [Proof of Authority Module](#proof-of-authority-module)
+    - [Module Functionality](#module-functionality-1)
+      - [Validator Management](#validator-management)
+      - [Staking Parameters Update](#staking-parameters-update)
+      - [Administrative Rights](#administrative-rights)
+      - [Commands](#commands-1)
+        - [Update Parameters (update-params)](#update-parameters-update-params-1)
+        - [Update Staking Parameters (update-staking-params)](#update-staking-parameters-update-staking-params)
+        - [Set Voting Power (set-power)](#set-voting-power-set-power)
+        - [Remove Pending Validator (remove-pending)](#remove-pending-validator-remove-pending)
+        - [Remove Validator (remove)](#remove-validator-remove)
+  - [Token Factory Module](#token-factory-module)
+    - [Module Functionality](#module-functionality-2)
+      - [Token Minting](#token-minting)
+      - [Token Burning](#token-burning)
+      - [Token Administration](#token-administration)
+      - [Metadata Management](#metadata-management)
+      - [Commands](#commands-2)
+        - [Burn (burn)](#burn-burn)
+        - [Burn From (burn-from)](#burn-from-burn-from)
+        - [Mint (mint)](#mint-mint)
+        - [Change Admin (change-admin)](#change-admin-change-admin)
+        - [Create Denom (create-denom)](#create-denom-create-denom)
+        - [Force Transfer (force-transfer)](#force-transfer-force-transfer)
+        - [Modify Metadata (modify-metadata)](#modify-metadata-modify-metadata)
+
+## Manifest Module
+
+The Manifest module is responsible for handling inflation and manual minting events. Below is a structured breakdown of its components and functionalities:
+
+### Module Functionality
+
+Stakeholder Management: Allows the PoA admin to designate stakeholders, who can be one or multiple manifest wallet addresses. These stakeholders are eligible to receive assets issued by the PoA admin.
+
+#### Asset Issuance:
+
+- Manual Issuance: The PoA admin can manually mint and disburse a specified amount of tokens to the stakeholders.
+
+- Automatic Inflation: When enabled, tokens are minted every block, aiming for a predetermined total over a year.
```

Comment:
predetermined total what? inflation? tokens?

Reply:
umfx tokens, set in the module params. Will add content here